var/spack/repos/builtin/packages/py-melissa-core/package.py

```diff
@@ -31,17 +33,22 @@ class PyMelissaCore(PythonPackage):
     depends_on("py-setuptools@46.4:", type=("build"))
     # requirements.txt (SA)
     depends_on("py-pyzmq@22.3.0:", type="run")
-    depends_on("py-mpi4py@3.1.3:", type="run")
+    depends_on("py-mpi4py@3.1.3", type="run")
     depends_on("py-numpy@1.21:", type="run")
     depends_on("py-jsonschema@4.5:", type="run")
     depends_on("py-python-rapidjson@1.8:", type="run")
     depends_on("py-scipy@1.10.0:", type="run")
-    depends_on("py-cloudpickle@2.2.0:", type="run")
+    depends_on("py-plotext@5.2.8:", type="run")
```

Comment:
Technically this should be:
```suggestion
    depends_on("py-plotext@5.2.8:", when="@sc23", type="run")
```
since it's only a dev dependency for other versions

Reply:
You are right. However we first moved plotext to the dev dependencies when we first created the `py-melissa-core` package (see [this discussion](https://github.com/spack/spack/pull/37609#discussion_r1198258492)). At that point the idea was to avoid creating a `py-plotext` package which I finally did. I think it's for the best if it remains a common dependencies.